tests/4.5/target_teams_distribute/test_target_teams_distribute_collapse.c

```diff
@@ -105,8 +105,14 @@ int test_collapse2() {
 }
 
 int main() {
+  
+  //Check for offloading
+  int is_offloading;
+  OMPVV_TEST_AND_SET_OFFLOADING(is_offloading);  
```

Comment:
Use OMPVV_TEST_OFFLOADING if you are not using the `is_offloading` for anything else.